actionpack/lib/action_controller/railtie.rb

```diff
@@ -67,5 +67,12 @@ class Railtie < Rails::Railtie #:nodoc:
         config.compile_methods! if config.respond_to?(:compile_methods!)
       end
     end
+
+    initializer "setup session store" do |app|
+      ActiveSupport.on_load(:action_controller) do
```

Comment:
`Rails::Application::Configuration` doesn't define any initializer. It should be in bootstrap or finisher. I believe finisher is the right place to put since we only want to set it if it was not already set.